From be8e104834df8b1aadec72f4896852ae746e9629 Mon Sep 17 00:00:00 2001
From: Doug Ledford <dledford@redhat.com>
Date: Wed, 24 Sep 2014 14:58:28 -0400
Subject: [infiniband] usnic: Remove '0x' when using %pa format

Message-id: <54ae68deed47cd975d9a79b2cc61443ee4ba7883.1411569661.git.dledford@redhat.com>
Patchwork-id: 96599
O-Subject: [Patch RHEL7 26/29] IB/usnic: Remove '0x' when using %pa format
Bugzilla: 916382
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

From: Fabio Estevam <fabio.estevam@freescale.com>

%pa format already prints in hexadecimal format, so remove the '0x' annotation
to avoid a double '0x0x' pattern.

Signed-off-by: Fabio Estevam <fabio.estevam@freescale.com>
Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit 970918b32b030e9b3966e5ccb5f4a5a5b515a5b1)
Signed-off-by: Doug Ledford <dledford@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/hw/usnic/usnic_uiom.c b/drivers/infiniband/hw/usnic/usnic_uiom.c
index 16755cd..801a1d6 100644
--- a/drivers/infiniband/hw/usnic/usnic_uiom.c
+++ b/drivers/infiniband/hw/usnic/usnic_uiom.c
@@ -286,7 +286,7 @@ iter_chunk:
     err = iommu_map(pd->domain, va_start, pa_start,
        size, flags);
     if (err) {
-     usnic_err("Failed to map va 0x%lx pa 0x%pa size 0x%zx with err %d\n",
+     usnic_err("Failed to map va 0x%lx pa %pa size 0x%zx with err %d\n",
       va_start, &pa_start, size, err);
      goto err_out;
     }
-- 
1.7.1